config/development.yml.erb

```diff
@@ -35,8 +35,9 @@ db_writer: 'mysql://root@localhost/'
 
 custom_error_response: false
 
-# Used for the exporting teacher enrollment data to a gsheet
-enrollments_summer_2020_gsheet_key:
+# Used for exporting the workshop data to a gsheet
+applications_2022_2023_gsheet_key: !Secret
+gdrive_export_secret: !Secret
 
 javabuilder_private_key: !Secret
 javabuilder_key_password: !Secret
```

Comment:
this is a bit off-topic, but this reminds me that there may be some secret keys in our developer setup process that could be added in this way, so that we don't have to manually add them to locals.yml. `properties_encryption_key` comes to mind for me: https://github.com/code-dot-org/code-dot-org/blob/31a934cf70a088ed1f74d42ee1b7c6038efb7e16/locals.yml.default#L53-L56

@megcrenshaw , are there any other shared keys which come to mind for you from your recent setup experience?

Reply:
Good call. `properties_encryption_key:` is the only key in my `locals` file right now, and it's already part of Secrets Manager. Shall I go ahead and add it to the development yml file?

Reply:
yes please, that would be great!

Reply:
could please also change the lines in locals.yml.default from
```
 # Code.org engineers should obtain this from AWS Secrets Manager at: 
 # https://console.aws.amazon.com/secretsmanager/home?region=us-east-1#/secret?name=development%2Fcdo%2Fproperties_encryption_key 
```
to something like this?
```
 # Code.org engineers with AWS credentials should get this automatically via AWS Secrets Manager.
```
the note about Contributors should probably stay.

Reply:
Sounds good!